### PR TITLE
USEID-434 Update design of the widget

### PR DESCRIPTION
**Pull request by pgurusinga**

### CODE REVIEW

File: src/main/kotlin/de/bund/digitalservice/useid/statics/WidgetController.kt [hunk 1]
```diff
@@ -9,4 +9,7 @@ import reactor.core.publisher.Mono
 class WidgetController {
     @GetMapping("/widget")
     fun widget(model: Model): Mono<String> = Mono.just("widget")
+
+    @GetMapping("/incompatible")
```
Comment (pgurusinga) [inline src/main/kotlin/de/bund/digitalservice/useid/statics/WidgetController.kt:13]: still unsure with the naming. `/not-supported` ?

Comment (hannesschaletzky) [reply src/main/kotlin/de/bund/digitalservice/useid/statics/WidgetController.kt:13]: /not-supported sounds a bit smoother. But really unsure about this as well

File: src/main/resources/static/css/widget.css [hunk 2]
```diff
@@ -119,9 +126,62 @@ h2 {
...
+    box-shadow:         0 0 0 4px #fff, 0 0 0 8px #004b76;
+}
 .identify h2 {
     margin: 0;
 }
 .widget {
     max-width: 400px;
 }
+
+/*INCOMPATIBLE PAGE*/
```
Comment (hannesschaletzky) [inline src/main/resources/static/css/widget.css:141]: Then you'd need to change it here and downwards as well.

**Review by hannesschaletzky — APPROVED** [review]
Looks good!

**Pull request merged**